generator/src/components/FormCheckbox/index.svelte

```diff
@@ -1,12 +1,15 @@
 <script lang="ts">
   import Checkbox from "$assets/check.svg?component"
   export let name: string
-  export let label: string = ""
+  export let label: string | undefined
+  export let value: boolean | undefined
```

Comment:
Why would we call this `value` instead of `checked`? Also why would we allow `undefined` for a boolean?

```suggestion
  export let checked: boolean = false
```